core/src/test/java/org/apache/iceberg/expressions/TestExpressionParser.java

```diff
@@ -20,16 +20,16 @@
 
 import static org.apache.iceberg.types.Types.NestedField.optional;
 import static org.apache.iceberg.types.Types.NestedField.required;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
```

Comment:
I believe this should be `Assertions.assertThatThrownBy`. 
Also as I mentioned on the other PR it's ok to combine different packages as long as the PR doesn't get too big

Reply:
Thanks for the review, I'll merge the other PRs to reflect your review and I'll change it back to the original `Assertions.assertThatThrownBy`.